blueflood-core/pom.xml

```diff
@@ -101,6 +101,7 @@
               <!-- do not require zookeeper for these tests. -->
               <argLine>-DZOOKEEPER_CLUSTER=NONE</argLine>
               <argLine>-Dlog4j.configuration=file://${basedir}/src/test/resources/log4j-test.properties</argLine>
+              <argLine>${jacoco.agent.it.argLine}</argLine>
```

Comment:
The parent POM https://github.com/rackerlabs/blueflood/pull/420/files#diff-600376dffeb79835ede4a0b285078036R166